indico/modules/rb_new/operations/rooms.py

```diff
@@ -137,10 +137,11 @@ def get_room_statistics(room):
     return data
 
 
-def search_for_rooms(filters, is_admin_override, availability=None):
+def search_for_rooms(filters, admin_override_enabled, availability=None):
     """Search for a room, using the provided filters.
 
     :param filters: The filters, provided as a dictionary
+    :param admin_override_enabled: A boolean specifying whether admins have override privilages
```

Comment:
I'd call the param `allow_admin` like we use everywhere else in the backend (e.g. in can_*) and give it a default value

Reply:
Should the default value be False?

Reply:
That's a good question... generally we have `allow_admin=True` as the default for all access checks, since we usually don't have opt-in admin privs. But for this one I think `False` makes more sense (and it's not a typical access check anyway).